### DISCUSSION TITLE
Hello everyone

### DISCUSSION THREAD

**Discussion by Andrea-Bruno** [cheshire-cat-ai/core#998] [General]
twenty years ago I did a very similar project, a programmable AI chat, which was actually a plugin of a larger project that was published here:
https://sourceforge.net/projects/cmsaspnet/
I abandoned the project because everyone told me that people will never chat with robots..
I would like to contribute to the realization of the project, so I offer my availability.

**Comment by pieroit** [comment]
welcome @Andrea-Bruno !
